go.mod

```diff
@@ -109,7 +109,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/apiextensions-apiserver v0.31.1 // indirect
 	k8s.io/cloud-provider v0.30.1 // indirect
-	k8s.io/component-helpers v0.30.1 // indirect
+	k8s.io/component-helpers v0.31.1 // indirect
```

Comment:
I assume we are doing this due to a incompatibility between go 1.22 and 1.23 and we'll revert this back to during actual 1.32 bump?